docs/wikidata-game.md

```diff
@@ -0,0 +1,30 @@
+# Introduction
+In order to measure the quality fo the pipeline, and to add references to Wikidata we implemented a version of the distributed game.
```

Comment:
"of"... Quick question, do you mind if we simply made proofreading commits to each other's doc branches? 

It seems to me more viable than flooding each other's reviews with comments about spelling mistakes.

Reply:
Go for it!